packages/web-components/fast-foundation/src/tree-view/tree-view.template.ts

```diff
@@ -14,6 +14,7 @@ export const TreeViewTemplate = html<TreeView>`
         @focus="${(x, c) => x.handleFocus(c.event as FocusEvent)}"
         @blur="${(x, c) => x.handleBlur(c.event as FocusEvent)}"
     >
-        <slot ${slotted("slottedTreeItems")}></slot>
+        <slot></slot>
+        <slot name="item" ${slotted("slottedTreeItems")}></slot>
```

Comment:
It seems like we would have all default items appearing before the slotted named items, what scenario are we trying to allow by doing this?